lib/algo/DMD.h

```diff
@@ -156,8 +166,13 @@ class DMD
      * @brief Constructor.
      *
      * @param[in] dim        The full-order state dimension.
+     * @param[in] init_os_s  Use initial state as offset.
+     * @param[in] init_os_d  Use initial derivative as offset.
+     * @param[in] mean_os_s  Use state mean as offset.
+     * @param[in] mean_os_d  Use derivative mean as offset.
```

Comment:
It seems a state offset and a derivative offset can be used together. This should be explained in a comment.

Reply:
This is now replaced by a general offset option.